sample_data_config/crossref-event/crossref-event-data-pipeline.config.yaml

```diff
@@ -18,5 +18,5 @@ schemaFile:
 # the bigquery data warehouse
 publisherIdPrefixes:
   - '10.7554'
-crossrefEventBaseUrl: "https://api.eventdata.crossref.org/v1/events?rows=1000"
+crossrefEventBaseUrl: 'https://api.eventdata.crossref.org/v1/events?rows=100&mailto=h.ciplak@elifesciences.org'
```

Comment:
just mentioning that this repo is public, not sure if email crawlers will find it (and if you mind it). But maybe not.